scripts/tests/test_ecs_spec.py

```diff
@@ -1,13 +1,14 @@
 import os
 import sys
+import glob
 import unittest
 
 sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
 
 from scripts import schema_reader
 
 
-(nested, flat) = schema_reader.load_ecs()
+(nested, flat) = schema_reader.load_ecs(sorted(glob.glob('../../schemas/*.yml')))
```

Comment:
Hey @mcpate, the tests are still failing because the glob path is relative to where the script is invoked from, not from the script file itself.

What I'll propose here is not perfect, but I can clean up the code around this later.

For now, if you just apply the following change, the tests will pass and I'll merge.

```suggestion
(nested, flat) = schema_reader.load_ecs(sorted(glob.glob('schemas/*.yml')))
```